api/lib/opentelemetry/instrumentation/base.rb

```diff
@@ -59,9 +59,17 @@ module Instrumentation
     # convention for environment variable name is the library name, upcased with
     # '::' replaced by underscores, OPENTELEMETRY shortened to OTEL_{LANG}, and '_ENABLED' appended.
     # For example: OTEL_RUBY_INSTRUMENTATION_SINATRA_ENABLED = false.
-    class Base
+    class Base # rubocop:disable Metrics/ClassLength
       class << self
         NAME_REGEX = /^(?:(?<namespace>[a-zA-Z0-9_:]+):{2})?(?<classname>[a-zA-Z0-9_]+)$/.freeze
+        VALIDATORS = {
+          array: ->(v) { v.is_a?(Array) },
+          boolean: ->(v) { v.is_a?(TrueClass) || v.is_a?(FalseClass) },
```

Comment:
_sigh_ I almost think it's worth adding a `# rubocop:disable` to this line, but not quite 🤷